
High_center: [0.5 , 0.0, 0.3]


BH1_S: [ 0.6872160179268519, -0.14076322774510802, 0.022745403230982633]

BH2_S: [0.617426846065691 , -0.139827712624430  , 0.022430527407691]

BH3_S: [0.547637674204531 , -0.138892197503752  , 0.022115651584400]

BH4_S: [0.47784850234337045, -0.13795668238307343, 0.02180077576110917]

BH1_G: [0.4486985001162725, 0.22299845949006336, 0.02463288252981005]

BH2_G: [0.5447036303651978, 0.1477191846817073, 0.025160634657607883 ]

BH3_G: [0.46847657797357173, 0.04301584096398785, 0.02372385873126041]

BH4_G: [0.669827234358595, 0.18942833873609624, 0.026672426228587645]    
